experimental/traffic-portal/src/app/api/testing/profile.service.ts

```diff
@@ -282,12 +282,12 @@ export class ProfileService {
 	public async createParameter(parameter: RequestParameter): Promise<ResponseParameter> {
 		const t = {
 			...parameter,
-			configFile: "cfg.txt",
+			configFile: parameter.configFile,
 			id: ++this.lastParamID,
 			lastUpdated: new Date(),
 			profiles: [],
-			secure: false,
-			value: "100"
+			secure: parameter.secure,
+			value: parameter.value ?? ""
 		};
```

Comment:
So, I think you're misunderstanding what this is doing and you copy/pasted it. `...` is an expansion, with arrays it lays out each member as an argument, with objects it does the same thing with properties. For example:
![image](https://github.com/apache/trafficcontrol/assets/6013378/9e2fbb14-6387-4baf-a71b-b1bd1a800e06)

So you don't need the secure, value, or configFile properties at all.

Reply:
Thanks for explaining that. However, when I remove the `value` property, it doesn't compile, possibly because of the types being different for the `value` attribute in `RequestParameter` (string | null) vs `ResponseParameter` (string). Any idea how to resolve that?